ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -81,9 +81,9 @@ private class JdbcLedgerDao(
 
   private val SQL_SELECT_LEDGER_ID = SQL("select ledger_id from parameters")
 
-  override def lookupLedgerId(): Future[Option[LedgerId]] =
+  override def lookupLedgerId(callerId: String): Future[Option[LedgerId]] =
     dbDispatcher
-      .executeSql("get_ledger_id") { implicit conn =>
+      .executeSql(s"get_ledger_id$callerId") { implicit conn =>
```

Comment:
What purpose does the `callerId` serve?

Reply:
was using this for some initialization instrumentation : now removed